pkg/reconciler/clusterimagepolicy/clusterimagepolicy.go

```diff
@@ -90,28 +97,40 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, cip *v1alpha1.ClusterIma
 	if err != nil {
 		if !apierrs.IsNotFound(err) {
 			logging.FromContext(ctx).Errorf("Failed to get configmap: %v", err)
+			cip.Status.MarkCMUpdateFailed(err.Error())
```

Comment:
Should we add a status for the url's failure or success operation when fetching a policy ?

Reply:
I thought we wanted to give more descriptive errors about what it could go wrong with a policy.